packages/engine-core/src/binary/BinaryEngine.ts

```diff
@@ -844,6 +845,26 @@ You very likely have the wrong "binaryTarget" defined in the schema.prisma file.
     return JSON.parse(result.stdout)
   }
 
+  async getDmmf(): Promise<DMMF.Document> {
+    if (!this.getDmmfPromise) {
+      this.getDmmfPromise = this._getDmmf()
+    }
+    return this.getDmmfPromise
+  }
+
+  private async _getDmmf(): Promise<DMMF.Document> {
+    const prismaPath = await this.getPrismaPath()
+
+    const env = await this.getEngineEnvVars()
+
+    const result = await execa(prismaPath, ['cli', 'dmmf'], {
```

Comment:
```suggestion
    const result = await execa(prismaPath, ['--enable-raw-queries', 'cli', 'dmmf'], {
```

This is needed for it to work.

Reply:
Thank you!